code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -62,7 +62,7 @@
 
 /obj/item/organ/body_egg/changeling_egg/egg_process(delta_time, times_fired)
 	// Changeling eggs grow in dead people
-	time += delta_time
+	time += delta_time * 10
```

Comment:
```suggestion
	time += delta_time SECONDS
```

This should probably use the `SECONDS` define.

Reply:
As far as I'm aware we don't use the SECONDS macro for non-constant values.

Similarly, using `delta_time SECONDS` gives off the wrong impression from reading the code. It implies that delta_time isn't in seconds. delta_time **is** in seconds. What the SECONDS macro does, counter-intuitively, is convert a given value to deciseconds.